scalable-go-service/index.md

```diff
@@ -0,0 +1,322 @@
+#  Writing Scalable Go Services
+
+## Problem
+
+How do we show the customer a complete set of relevant hotel rates before they
+lose interest and close the tab? Despite trying various strategies to address
+this, the team will likely continue to face this challenge in the future as user
+expectations on speed and accuracy grow. The bottom line is, we need to load a
+lot of rates from many different vendors to show that we have as complete of an
+inventory as any other travel site. And we need to do this fast.
+
+Two of our main hotel vendors have announced major changes to the APIs we use to
```

Comment:
This is a better introductory paragraph. It states the explicit motivation that is driving the timing of the change and the value of this work. Then you can use the previous paragraph ("How do we show the customer...") to show the ongoing concerns that drive specific technical challenges of this work.